pkg/console/operator/sync_v400.go

```diff
@@ -367,6 +368,12 @@ func (co *consoleOperator) SyncConfigMap(
 		return nil, false, "FailedGetManagedConfig", mcErr
 	}
 
+	nodeList, nodeListErr := co.nodeClient.Nodes().List(ctx, metav1.ListOptions{})
+	if nodeListErr != nil {
+		return nil, false, "FailedListNodes", nodeListErr
+	}
+	nodeArchitectures := getNodeArchitectures(nodeList)
```

Comment:
how often will this be updated? anytime there is an additional node added to the system ?